skops/io/_sklearn.py

```diff
@@ -214,8 +220,24 @@ def __init__(
 
 
 def loss_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
-    state = reduce_get_state(obj, save_context)
-    state["__loader__"] = "LossNode"
+    reduce = obj.__reduce__()
```

Comment:
For now we are running this code for known objects, and since Cython is a build dependency and the installed cython doesn't matter, and that we use modern cython in sklearn build, we're fine.

But if we start supporting arbitrary objects via reduce for cython, then this would be an issue.